### PR TITLE
[feature/cumulative--10.0.0] AB#33500 - Closed Button Gap

### PR DESCRIPTION
**Pull request by alejosod**
Button changes:
* Close Gaps
    * Need update the outline prop to extend outline and text colors - done
        * Change the “outlined” prop name to “outline” - done
    * Update color prop to use the theme.palette colors. - done
        * Color
            * Alert
                * Keep
            * Alternate
                * Keep the current color and rename to “Secondary”.
                * Delete Secondary style
            * Disable
                * Keep
            * Light
                * Delete
            * Outline
                * Delete
            * Primary
                * Keep
            * Secondary
                * Delete
            * Success
                * Keep
            * Transparent
                * Delete and create prop to handle the styling
            * Warning
                * Keep
        * Inverse
            * Delete the colors from that we are deleting form the color prop’s enum
        * Outlined
            * Change to outline.
            * Change label color to default black for all outline styles
            * Apple the changes being done int he color prop.
    * Need Touchscreen styling. - done
        * Add new boolean prop called “pill”.
        * Using the withStyles HOC style it.
* SCSS has already been converted to JSS - done
* Update the documentation page with the new code. - done
    * A good example ./docs/src/inputs/durationPicker/durationPicker.jsx

### CODE REVIEW

**Comment by morethanfire** [comment]
@alejosod, it looks like there is a unit test check failing. Can you please commit a fix?

image

File: src/inputs/button/button.jsx
Comment (morethanfire) [inline src/inputs/button/button.jsx:328]: Please remove `disabled` for all version 10.0.0 gap closures. It was a prop that was deprecated in version 9.

File: src/inputs/button/button.jsx
Comment (morethanfire) [inline src/inputs/button/button.jsx:349]: I think that we should be able to have an outline and for the button to be disabled.

File: src/inputs/button/button.jsx
Comment (morethanfire) [inline src/inputs/button/button.jsx:344]: I think that we should be able to have a transparent button and for the button to be disabled.

File: src/inputs/button/__test__/button.test.js
Comment (morethanfire) [inline src/inputs/button/__test__/button.test.js:16]: Please test more than if Button renders. We can test the classes, events, etc.

File: docs/src/inputs/button/buttonPillExample.jsx
Comment (morethanfire) [inline docs/src/inputs/button/buttonPillExample.jsx:13]: Nice!

File: docs/src/inputs/button/buttonOutlineExample.jsx
Comment (morethanfire) [inline docs/src/inputs/button/buttonOutlineExample.jsx:50]: I like it.

File: docs/src/inputs/button/button.jsx
Comment (morethanfire) [inline docs/src/inputs/button/button.jsx:206]: Please use the new documentation components. We can get on a call to over this. Ping me when you can.

**Review by morethanfire — CHANGES_REQUESTED** [review]
@alejosod, the PR is coming along. There a few changes I mentioned that need to take place and I have some concerns about what `.idea/` is and why we are introducing that to the repo

File: .gitignore
Comment (alejosod) [reply .gitignore:11]: They are configuration for my IDE

**Comment by groberts314** [comment]
PR Review Checkpoint Meeting Thurs 2020-09-17.
* @alejosod ... Looks like a unit test is maybe failing?  Please check https://dev.azure.com/saddlebackchurch/b8a0d0bb-a4fe-48e3-9a60-2f1440145fc4/_build/results?buildId=20996
* @morethanfire Looks like some work was done to address your change requests.  Can you re-review?

**Comment by groberts314** [comment]
@alejosod @morethanfire Is this PR targeting `dev` correct?  Or should it be targeting `feature/cumulative--10.0.0`?

**Comment by morethanfire** [comment]
Heads up @alejosod, I am going to start working on this branch to finish where you left off. 

This PR cannot be merged until PR #222 is merged. Probably, should wait on reviewing it until it is merged as well.

**Pull request merged**
